components/cap-contributor-list.js

```diff
@@ -0,0 +1,92 @@
+import { LitElement, html, css } from "../lib/lit.js";
+import { baseStyles } from "../lib/wc-base.js";
+import { contributorLinks } from "../data/contributors.js";
+
+export class CapContributorList extends LitElement {
+	static properties = {
+		decoration: { type: String | undefined },
+		theme: { type: String | undefined },
+	};
+
+	static styles = [
+		baseStyles,
+		css`
+			:host {
+				--color-link: var(--color-blue-400);
+				--color-link-hover: var(--color-gray-600);
+				--color-publisher: var(--color-gray-500);
+				--color-date: var(--color-gray-400);
+				--list-style-type: none;
+
+				font-family: var(--font-sans-text);
+			}
```

Comment:
Super cool thing about css variables — you can define them locally in `:root` (for a page) or `:host` (for scoped web components) and then reuse that variable anywhere else within that scoped component or page, or later reassign them depending on your UI theming needs.